ios/http/FileWebServer.m

```diff
@@ -0,0 +1,111 @@
+//
+//  WebServer.m
+//  PPSSPP
+//
+//  Created by Daniel Gillespie on 7/25/15.
+//
+
+#import "FileWebServer.h"
+
+@implementation FileWebServer
+
+-(id)init {
+    webServer = [[GCDWebUploader alloc] initWithUploadDirectory: [self getDocumentDirectory]];
+    webServer.delegate = self;
+    webServer.allowHiddenItems = NO;
+    
+    return self;
```

Comment:
This is incorrect, because you are returning `self` even though it has not been initialized yet. It's important to call `[super init]`
```objective-c
+ (instancetype)init {
    self = [super init];
    if (self == nil) {
        return nil;
    }
    
    webServer = [[GCDWebUploader alloc] initWithUploadDirectory: [self getDocumentDirectory]];
    webServer.delegate = self;
    webServer.allowHiddenItems = NO;
    
    return self;
}
```